### DISCUSSION TITLE
Get "Activating the Pull Requests and Issues extension failed. Please make sure you have git installed." on default Codespace configuration.

### DISCUSSION THREAD

**Discussion by DevCas1** [community/community#17163] [Codespaces]
Title says it all:
Whenever I open a Codespace on an HTML, CSS and JS repository with a default configuration, I get this error message "Activating the Pull Requests and Issues extension failed. Please make sure you have git installed." and see that the Pull Requests and Issues extension is loaded but not accessible.

Haven't been able to figure out how to fix this, anyone any idea?

**Comment by legomushroom** [comment]
@DevCas1 thanks for reporting 👍 Mind sharing a repo you creating this Codespace from?

**Reply by DevCas1** [reply]
@legomushroom The private repo I tried this with is DevCas1.github.io.
If you'd like I could share a .zip of the repo?
Don't have any codespace specific files on it, created a codespace with the default template (as it's the only thing I have access to, using it through beta).
Codspace info:
- **Branch:** development
- **Dev container configuration:** Default Codespaces configuration
- **Region:** Europe West
- **Machine type:** 4-core • 8GB RAM  • 32GB storage

**Comment by DevCas1** [comment]
When I open a Bash terminal in the Codespace and type `$ git` I get the following response: 
`usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]`
This tells me Git is most definitely installed on the Codespace, and the GitHub extension simply fails to start (becuase it can't find / reach Git).
